src/driver/react-native/ReactNativeConnectionOptions.ts

```diff
@@ -19,4 +19,9 @@ export interface ReactNativeConnectionOptions extends BaseConnectionOptions {
      * Storage Location
      */
     readonly location: string;
+
+    /**
+     * Driver
```

Comment:
I think here we need to provide a better description of this option. Also maybe it worth document it in the docs (for website)